main.go

```diff
@@ -995,20 +1020,91 @@ func getPeerText(ctx context.Context) string {
 		peerCNTreachable := peerCount - peerStats.CNT0
 		if peerCNTreachable > 0 {
 			peerStats.RTTAVG = peerStats.RTTSUM / peerCNTreachable
-			peerStats.PCT1 = peerStats.CNT1 / peerCNTreachable * 100
-			peerStats.PCT1items = peerStats.PCT1 * granularitySmall / 100
-			peerStats.PCT2 = peerStats.CNT2 / peerCNTreachable * 100
-			peerStats.PCT2items = peerStats.PCT2 * granularitySmall / 100
-			peerStats.PCT3 = peerStats.CNT3 / peerCNTreachable * 100
-			peerStats.PCT3items = peerStats.PCT3 * granularitySmall / 100
-			peerStats.PCT4 = peerStats.CNT4 / peerCNTreachable * 100
-			peerStats.PCT4items = peerStats.PCT4 * granularitySmall / 100
+			peerStats.PCT1 = float32(peerStats.CNT1) / float32(peerCNTreachable) * 100
+			peerStats.PCT1items = int(peerStats.PCT1) * granularitySmall / 100
+			peerStats.PCT2 = float32(peerStats.CNT2) / float32(peerCNTreachable) * 100
+			peerStats.PCT2items = int(peerStats.PCT2) * granularitySmall / 100
+			peerStats.PCT3 = float32(peerStats.CNT3) / float32(peerCNTreachable) * 100
+			peerStats.PCT3items = int(peerStats.PCT3) * granularitySmall / 100
+			peerStats.PCT4 = float32(peerStats.CNT4) / float32(peerCNTreachable) * 100
+			peerStats.PCT4items = int(peerStats.PCT4) * granularitySmall / 100
 		}
 		// TODO: lookup geoIP data
 		sb.WriteString(fmt.Sprintf(" [yellow]%-46s[white]\n", "Peer analysis done!"))
 		peerAnalysisDate = uint64(time.Now().Unix() - 1)
+		checkPeers = true
+	} else if showPeers {
+		sb.WriteString("       RTT : Peers / Percent\n")
+		sb.WriteString(fmt.Sprintf(
+			"    0-50ms : [blue]%5s[white]   [blue]%.f[white]%%",
+			strconv.Itoa(peerStats.CNT1),
+			peerStats.PCT1,
+		))
+		sb.WriteString(fmt.Sprintf(
+			"%"+strconv.Itoa(5-len(fmt.Sprintf("%.f", peerStats.PCT1)))+"s",
+			" ",
+		))
+		for i := 0; i <= granularitySmall-1; i++ {
```

Comment:
You should be able to simplify the condition to just `i < granularitySmall`

Reply:
That's fair. I'm moving things from shell to go, so it's sometimes easier to not optimize. I'll pick that up on my next pass.